src/constants/translations/en/my-resources-page.json

```diff
@@ -1,19 +1,26 @@
 {
   "tabs": {
     "lessons": "Lessons",
-    "tests": "Tests",
+    "quizzes": "Quizzes",
     "attachments": "Attachments"
   },
   "lessons": {
-    "newLessonBtn":"New lesson",
+    "newLessonBtn": "New lesson",
```

Comment:
Do you need this space

Reply:
It is formatted VS Code, and this space between the key and value